christfellowshipchurch/web-app-v2
# Share sheet button

### About
This PR implements the iOS Share API.

### Test Instructions
1. Open on a mobile device and go to any page with a share or download button, for example `/articles/new-summer-wallpaper`, test downloading and sharing. 
2. Also test these features on a different device with a bigger screen to ensure that it still allows to share and download on those devices.
3. If possible test on Android as well.

### Screenshots
<img width="1440" alt="Screenshot 2023-10-02 at 2 49 45 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/94265294/2aa6d645-b4b4-4bf4-80e1-148c7aa9cf2f">

### Closes Tickets
[CFDP-2180](https://christfellowshipchurch.atlassian.net/browse/CFDP-2180)

[CFDP-2180]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2180?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
